test/testcafe/spec/ChallengeAuthenticatorPassword_spec.js

```diff
@@ -79,7 +79,7 @@ test.requestHooks(mockInvalidPassword)('challege password authenticator with inv
   await challengePasswordPage.verifyFactor('credentials.passcode', 'test');
   await challengePasswordPage.clickNextButton();
 
-  await t.expect(challengePasswordPage.getPasswordFieldErrorMessage()).contains('The passcode is absent or invalid');
+  await t.expect(challengePasswordPage.getInvalidOTPError()).contains('You do not have permission to perform the requested action.');
```

Comment:
why did we have to update it ?

Reply:
We removed the error-answer-passcode-invalid I checked with the current IDX org, it returns the general error `You do not have permission to perform the requested action.`